### PR TITLE
68835: Document new Time Machine option for SMB shares.

### PR DESCRIPTION
**Pull request by Mrt134**
- Add entry about new checkbox in intro.rst.
sharing.rst:
- Add subsection to break up Wizard instructions and configuring Time Machine backup instructions.
- Clean up some extra admonition boxes in the Windows (SMB) Shares introduction.
- Rebuild "SMB Share Options" table. Numerous small fixes to entries in the table.
- Add entry for Time Machine to "SMB Share Options" table.
- HTML build testing: no issues.

### CODE REVIEW

File: userguide/sharing.rst [hunk 1]
```diff
@@ -1061,88 +1060,80 @@ provides more details for each configurable option.
...
+   +-------------------------+--------------+-------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+   | Use as home share       | checkbox     |             | Set to allow this share to hold user home directories. Only one share can be the home share. Note that lower case names for user home directories |
+   |                         |              |             | are strongly recommended, as Samba maps usernames to all lower case. For example, the username John will be mapped to a home directory named      |
+   |                         |              |             | :file:`john`. If the :guilabel:`Path` to the home share includes an upper case username, delete the existing user and recreate it in              |
+   |                         |              |             | :menuselection:`Accounts --> Users` with an all lower case :guilabel:`Username`. Return to :menuselection:`Sharing --> SMB` to create the home    |
```
Comment (a-stjohn) [inline userguide/sharing.rst]: Perhaps put an internal :ref: to the Accounts --> Users section of the guide? Just a thought.

File: userguide/sharing.rst [hunk 2]
```diff
@@ -1004,36 +863,25 @@ some troubleshooting tips. Reading through this entire chapter before
...
-   videos clarify setting up permissions on SMB shares. Another
-   helpful reference is
-   `Methods For Fine-Tuning Samba Permissions
-   <https://forums.freenas.org/index.php?threads/methods-for-fine-tuning-samba-permissions.50739/>`__.
-
-
-.. tip:: Run :command:`smbstatus` from the :ref:`Shell` for a list of
-   active connections and users.
+.. warning:: SMB1 is disabled by default for security. If legacy
+   clients are unable to connect to the share, go to
```
Comment (wblock) [inline userguide/sharing.rst]: Replace all of this with "If necessary, SMB1 can be enabled in Services --> SMB Settings."
Remove the rest.

File: userguide/sharing.rst [hunk 3]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+Creating Authenticated and Time Machine Shares
+----------------------------------------------
+
+macOS includes the Time Machine feature which can be used to perform
+automatic back ups. %brand% supports Time Machine backups for both
+:ref:`AFP <Apple (AFP) Shares>` and :ref:`SMB <Windows (SMB) Shares>`
+shares. This section has instructions to create Time Machine AFP and SMB
+shares, using the
+:menuselection:`Wizard`
+to create an AFP Time Machine share, configuring a Time Machine share.
```
Comment (wblock) [inline userguide/sharing.rst]: This sentence gets weird at the end.

File: userguide/sharing.rst [hunk 4]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+
+.. index:: Time Machine
+.. _Creating Authenticated and Time Machine Shares:
+
+Creating Authenticated and Time Machine Shares
+----------------------------------------------
+
+macOS includes the Time Machine feature which can be used to perform
+automatic back ups. %brand% supports Time Machine backups for both
+:ref:`AFP <Apple (AFP) Shares>` and :ref:`SMB <Windows (SMB) Shares>`
```
Comment (wblock) [inline userguide/sharing.rst]: Mention SMB first each time.

File: userguide/sharing.rst [hunk 5]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+shares. This section has instructions to create Time Machine AFP and SMB
```
Comment (wblock) [inline userguide/sharing.rst]: Mention SMB first.

File: userguide/sharing.rst [hunk 6]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+:menuselection:`Wizard`
+to create an AFP Time Machine share, configuring a Time Machine share.
+The process for creating an authenticated share for a user is the same
+as creating a Time Machine share for that user.
+
+
+Manual Creation of Authenticated or Time Machine Shares
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Both Time Machine and authenticated shares should be created on an
```
Comment (wblock) [inline userguide/sharing.rst]: Make this imperative.
"Create Time Machine or authenticated shares on a new dataset."

File: userguide/sharing.rst [hunk 7]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+:ref:`new dataset <Create Dataset>`.
+
+After the dataset is created, go to
```
Comment (wblock) [inline userguide/sharing.rst]:
```suggestion
Change permissions on the new dataset by going to
```

File: userguide/sharing.rst [hunk 8]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+:menuselection:`Storage --> Volumes`,
```
Comment (wblock) [inline userguide/sharing.rst]:
```suggestion
:menuselection:`Storage --> Volumes`.
```

File: userguide/sharing.rst [hunk 9]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+highlight the dataset, and click :guilabel:`Change Permissions`.
```
Comment (wblock) [inline userguide/sharing.rst]:
```suggestion
Select the dataset and click :guilabel:`Change Permissions`.
```

File: userguide/sharing.rst [hunk 10]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+   See :ref:`users <Users>` for more information.
+
+#. **Owner (group):** Select the desired group name. If the group does
+   not yet exist on the %brand% system, create one with
+   :menuselection:`Account --> Groups`.
+   See :ref:`groups <Groups>` for more information.
+
+#. Click :guilabel:`Change`.
+
+To create an authenticated or Time Machine share:
```
Comment (wblock) [inline userguide/sharing.rst]:
```suggestion
Create the authenticated or Time Machine share:
```

File: userguide/sharing.rst [hunk 11]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+   both a new *user1* user and a new *tm_backups* group will be
+   created. Since a new user is being created, this screen prompts for
+   the user password to be used when accessing the share. It also
+   provides an opportunity to change the default permissions on the
+   share. When finished, click :guilabel:`Return` to return to the
+   screen shown in
+   :numref:`Figure %s <create_time_machine_share_fig>`.
+
+#. Click the :guilabel:`Add` button.
+   **Remember to do this or the share will not be created**.
```
Comment (wblock) [inline userguide/sharing.rst]: I don't think this is necessary.
```suggestion
```

File: userguide/sharing.rst [hunk 12]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+   Clicking the :guilabel:`Add` button adds an entry to the
```
Comment (wblock) [inline userguide/sharing.rst]: This sentence also seems unnecessary.

File: userguide/sharing.rst [hunk 13]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+   :guilabel:`Name` frame with the name that was entered in
+   :guilabel:`Share name`.
+
+
+To configure multiple authenticated or Time Machine shares, repeat for
```
Comment (wblock) [inline userguide/sharing.rst]:
```suggestion
When creating multiple authenticated or Time Machine shares, repeat this process for
```

File: userguide/sharing.rst [hunk 14]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+each user, giving each user their own :guilabel:`Share name` and
```
Comment (wblock) [inline userguide/sharing.rst]:
```suggestion
each user. Give each user their own :guilabel:`Share name` and
```

File: userguide/sharing.rst [hunk 15]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+:guilabel:`Ownership`. When finished, click the :guilabel:`Next`
+button twice, then the :guilabel:`Confirm` button to create the
+shares. The Wizard automatically creates a dataset for each share with
```
Comment (wblock) [inline userguide/sharing.rst]:
```suggestion
shares. The Wizard creates a dataset for each share with
```

File: userguide/sharing.rst [hunk 16]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+.. _Configuring Time Machine Backups:
+
+Configuring Time Machine Backups
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Configuring a quota for each Time Machine share helps prevent backups
+from using all available space on the %brand% system. Time Machine waits
+two minutes before creating a full backup. Incremental backups are
+created each hour for the next 24 hours. After that, one backup is
+created each day, each week, and each month. Each of these hourly,
```
Comment (wblock) [inline userguide/sharing.rst]: Please verify the explanation of backups with Andrew or Bonnie.

File: userguide/sharing.rst [hunk 17]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+make sure that the quota size is large enough to hold the desired number
+of backups.** Note that a default installation of macOS is over 20 GiB.
+
+Configure a global quota using the instructions in
+`Set up Time Machine for multiple machines with OSX Server-Style Quotas
+<https://forums.freenas.org/index.php?threads/how-to-set-up-time-machine-for-multiple-machines-with-osx-server-style-quotas.47173/>`__.
+
+To configure a quota, go to
+:menuselection:`Storage --> Volumes`
+and highlight the share dataset. In the example shown in
```
Comment (wblock) [inline userguide/sharing.rst]:
```suggestion
and select the share dataset. In the example shown in
```

File: userguide/sharing.rst [hunk 18]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+
+.. _set_quota_fig:
+
+.. figure:: images/afp9a.png
+
+   Setting a Quota
+
+
+To configure Time Machine on the macOS client, go to
+:menuselection:`System Preferences --> Time Machine`
```
Comment (wblock) [inline userguide/sharing.rst]:
```suggestion
:menuselection:`System Preferences --> Time Machine`,
```

File: userguide/sharing.rst [hunk 19]
```diff
@@ -2697,3 +2541,202 @@ Go back to
...
+which opens the screen shown in
+:numref:`Figure %s <config_tm_osx>`.
+Click :guilabel:`ON` and a pop-up menu shows the %brand% system as a
+backup option. In this example, it is listed as
+*backup_user1 on "freenas"*. Highlight the %brand% system and click
+:guilabel:`Use Backup Disk`. A connection bar opens and prompts for
+the user account's password--in this example, the password that was
```
Comment (wblock) [inline userguide/sharing.rst]:
```suggestion
the user account's password. In this example, the password is the password that was
```

File: userguide/sharing.rst (hunk 16 above)
Comment (Mrt134) [reply userguide/sharing.rst]: Updated text with feedback from Andrew.

**Pull request merged**